lib/next_rails/bundle_report.rb

```diff
@@ -11,66 +11,9 @@ module BundleReport
     def compatibility(rails_version: nil, ruby_version: nil, include_rails_gems: nil)
```

Comment:
I feel it makes sense to split this method in 2:

```
def ruby_compatibility(ruby_version: nil)
  return unlees ruby_version

  puts RubyVersionCompatibility.new(options: { ruby_version: ruby_version }).generate
end

def rails_compatibility(rails_version: nil, include_rails_gems: nil)
  return unless rails_version

  puts RailsVersionCompatibility.new(options: { rails_version: rails_version, include_rails_gems: include_rails_gems }).generate
end
```

then update the binary to call the correct method as needed https://github.com/fastruby/next_rails/blob/7f7a9df5234aae5abab4bff1c287e30db02e99fb/exe/bundle_report#L70

Reply:
I updated this one and will continue updating the other good suggestions/fixes that you proposed. Thanks, Ariel!